src/Microsoft.AspNet.Mvc.Core/BindAttribute.cs

```diff
@@ -40,7 +41,8 @@ public BindAttribute(params string[] include)
         /// </param>
         public BindAttribute([NotNull] Type predicateProviderType)
         {
-            if (!typeof(IPropertyBindingPredicateProvider).IsAssignableFrom(predicateProviderType))
+            if (!typeof(IPropertyBindingPredicateProvider).GetTypeInfo()
+                    .IsAssignableFrom(predicateProviderType.GetTypeInfo()))
```

Comment:
It only seems that way because you made the change already :laughing:

It's pretty easy to paste the same 3 lines in all the files.